.github/workflows/ci-ubuntu.yml

```diff
@@ -73,13 +73,19 @@ jobs:
           ${{ env.CABAL_INSTALL }}
```

Comment:
You are installing `agda` with a `v1` command, would that work with a `v2` command?
You need to pass the `--installdir` then, though.

Reply:
This is a side effect of changing `install` to `v1-install`. In practice
I don't really care about how agda is installed as long as it's available
in `~/.cabal/bin` (which the previous `install --overwrite-policy=always`
guaranteed AFAICT).